broadinstitute/orsp-pub
# BTRX - 817 - Documents table sort

## Addresses
[BTRX-817](https://broadinstitute.atlassian.net/browse/BTRX-817): Sort by created date is not working as expected

## Changes
- Replace old Bootstrap Table with the new Bootstrap Table 2
- Some code optimization on imports
- On date sort column had to transform date String to date object in order to sort it as it was expected

## Testing
- Check Documents table is working in Project Review and Sample Data Cohort Review (download link, redirections, delete document, download document link)
- Check Associated project table stills working. This can be found in Sample Data Cohort Review, in Document's tab.
---
